command-expansion-server/tsconfig.json

```diff
@@ -98,6 +98,6 @@
     // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
     "skipLibCheck": true                                 /* Skip type checking all .d.ts files. */
   },
-  "exclude":["commanding_file_store"]
+  "exclude":["commanding_file_store", "build"]
```

Comment:
Just so I understand, was this added to exclude generated TS from being imported in this project?

Reply:
Yeah, so we don't typecheck the build directory